### PR TITLE
Updated tests for user accounts

### PR DESCRIPTION
**Pull request by edwardsbradbury**

### CODE REVIEW

**Review by LRichardson-git — APPROVED** [review]
tommy approved

File: src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java [hunk 1]
```diff
@@ -41,12 +53,39 @@ void testPostToCreateAccountWithValidData() {
...
     @Test
+    @DisplayName("Test PUT to updateAccount endpoint with valid data - responseCode should be 200 and message 'User updated!'")
+    void testPutToUpdateAccountWithValidData() {
+
+        userData = new HashMap<>();
+        userData.put("email", "ExampleEmail@example.com");
+        userData.put("password", "ExamplePassword");
+        userData.put("name", "Name");
+        userData.put("title", "Mr");
+        userData.put("birth_date", "10");
```
Comment (MarcinJakobik) [inline src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java:70]: birth_date->birth_day

File: src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java [hunk 2]
```diff
@@ -41,12 +53,39 @@ void testPostToCreateAccountWithValidData() {
...
+        userData.put("birth_month", "April");
+        userData.put("birth_year", "1990");
+        userData.put("firstname", "William");
```
Comment (MarcinJakobik) [inline src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java:73]: firstname->first_name

File: src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java [hunk 3]
```diff
@@ -41,12 +53,39 @@ void testPostToCreateAccountWithValidData() {
...
+        userData.put("lastname", "Bailey");
```
Comment (MarcinJakobik) [inline src/test/java/com/sparta/badgerBytes/tests/UserAccountDTOTests.java:74]: lastname->last_name

File: src/test/java/com/sparta/badgerBytes/testFramework/model/Injector.java [hunk 4]
```diff
@@ -29,6 +29,7 @@ public static <T extends SuperDTO> T deserialize(T dto, ConnectionManager.Method
                 response = ConnectionManager.getResponse(setMethod, setParams, setEndPoint);
             }
             jsonBody = response.body().toString();
+            System.out.println(jsonBody);
```
Comment (NicholasDaily) [inline src/test/java/com/sparta/badgerBytes/testFramework/model/Injector.java:32]: Could you remove this print statement

**Pull request closed without merging**